www/components/sidebar.tsx

```diff
@@ -262,6 +263,7 @@ export default function Sidebar({
                   className="block px-4 py-2 text-sm text-gray-700 dark:text-gray-200 hover:bg-gray-100 dark:hover:bg-gray-700 w-full text-left"
                   onClick={async () => {
                     await supabase.auth.signOut();
+                    clearSWRCache();
```

Comment:
When I logout my localStorage values are still there in app-cache

Reply:
<img width="1314" alt="image" src="https://github.com/user-attachments/assets/0885202f-fc72-4996-bc34-e4fde90be3ee" />